src/index.js

```diff
@@ -80,6 +81,17 @@ export const getCurrentNodeVersion = () => {
   return parseFloat(process.versions.node);
 };
 
+/**
+ * Returns the locally installed Electron version
+ * @return {string}
+ */
+export const getLocalElectronVersion = () => {
+  // Electron is not available in test environment so mock the version
+  if (process.env.NODE_ENV === "test") return "1.6.2";
```

Comment:
Yeah I'm going to remove that but the problem still exists, how can I mock the result of `getLocalElectronVersion` in test?

I'm calling it in `getTargets`. An option would be to inject it into `getTargets`?.